lib/ebpd_loader.cc

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+#include <string>
+#include "lib/ebpd_utils.h"
+#include "lib/ebpd_loader.h"
+#include "lib/ebpd.h"
+
+using namespace std;
+
+int
+EbpdLoader::LoadFile(const char *filepath, int ifindex) {
```

Comment:
If we're using C++, this should "const std::string& filepath" and then use filepath.c_str() later when passing to ebpd_load_xdp_prog().

Ideally we would use std::string_view, however std::string_view does not provide a way to return a null terminated string.

also make ifindex const. In general try to make all parameters const if there isn't a compelling reason not to.

Reply:
Done.